examples/Training/java/src/training/util/BytesConverter.java

```diff
@@ -1,71 +0,0 @@
-/*
```

Comment:
I don't think so. These classes are also part of the Gateway: https://github.com/openmicroscopy/openmicroscopy/tree/develop/components/blitz/src/omero/gateway/rnd . I think when I started the Gateway I copied them over, but just forgot to remove them from the training examples.

Reply:
they were for convenience
No need  to keep them in training now they are in the gateway

Reply:
an absolutely minor point since most people would use the relevant code bits from the example but the second argument for the Sudo instantiation in the main method should be sudoUsername